src/main.rs

```diff
@@ -44,8 +45,24 @@ pub fn main() {
 
     match builder_from_flags(bind_args.into_iter()) {
         Ok((builder, output)) => {
-            let mut bindings = builder.generate()
-                .expect("Unable to generate bindings");
+
+            let builder_result =
+                panic::catch_unwind(||
+                    builder.generate().expect("Unable to generate bindings")
+                );
+
+            if builder_result.is_err() {
```

Comment:
In the most recent changes, I kept ` if builder_result.is_err() {` on top because we would need this check to exit when bindings fail to generate. But I encapsulated the messages in an optional flag 😃